pkg/minishift/addon/manager/addon_manager.go

```diff
@@ -168,7 +168,7 @@ func (m *AddOnManager) String() string {
 }
 
 func (m *AddOnManager) applyAddOn(addOn addon.AddOn, context *command.ExecutionContext) error {
-	fmt.Print(fmt.Sprintf("Applying addon %s:", addOn.MetaData().Name()))
+	fmt.Print(fmt.Sprintf("-- Applying addon '%s':", addOn.MetaData().Name()))
```

Comment:
At the moment there is no guarantee that the `Name'` is unique (however, the folder names always are). Is it possible to at least include this? I am sure this will prevent issue hunting later, as the addons enabling does not happen on the 'name', but folder... right?